packages/components/src/FormField/FormField.test.tsx

```diff
@@ -270,6 +277,32 @@ describe("FormField", () => {
       });
     });
 
+    describe("when nested/structured", () => {
+      it("should display errors", async () => {
+        const { getByText } = render(
+          <Form onSubmit={jest.fn()}>
```

Comment:
I'm still investigating if there's a way to run the validations and produce an error without a `Form` component. Or maybe this test belongs somewhere else?

Reply:
Following up on this, I couldn't find a way to do this without `Form`. If we're uncomfortable with me importing `Form` here, I could make a 'fake' form using raw `react-hook-form` primitives that would serve the same purpose...

I'm not sure how independent we like our Atlantis components to be.